FreeRTOS/FreeRTOS
# Add makefile gcc build to mps2 qemu iar project

Description
-----------
Add a GCC build (makefile and Eclipse project that builds the makefile to the preexisting FreeRTOS/Demo/CORTEX_MPS2_QEMU_IAR project.  After this PR is merged the directory name needs changing to CORTEX_MPS2_QEMU_IAR_GCC, and the IAR specific files need moving to their own sub directory as the GCC build files already are.

Add batch files to start QEMU for both the IAR and GCC builds.

Add the small printf implementation printf-stdarg.c for use with the GCC build as the standard library uses too much stack.  The file was added to a common directory as it already appears in many demo specific directories.

Test Steps
-----------
The project in that directory runs a comprehensive test, so run both the IAR and GCC builds within that directory at all optimisation levels.



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
